src/how-to-build-and-run.md

```diff
@@ -80,6 +88,52 @@ compiling `rustc` is done in stages:
   can build the libraries with the stage2 compiler. The result ought
   to be identical to before, unless something has broken.
 
+
+#### Build Flags
+
+There are other flags you can pass to the build portion of x.py that can be 
+beneficial to cutting down compile times or fitting other things you might 
+need to change. They are:
+
+```bash
+Options:
+    -v, --verbose       use verbose output (-vv for very verbose)
+    -i, --incremental   use incremental compilation
+        --config FILE   TOML configuration file for build
+        --build BUILD   build target of the stage0 compiler
+        --host HOST     host targets to build
+        --target TARGET target targets to build
+        --on-fail CMD   command to run on failure
+        --stage N       stage to build
+        --keep-stage N  stage to keep without recompiling
+        --src DIR       path to the root of the rust checkout
+    -j, --jobs JOBS     number of jobs to run in parallel
+    -h, --help          print this help message
+```
+
+One thing to keep in mind is that `rustc` is a _bootstrapping_ compiler. That
```

Comment:
This is already discussed in the previous section. I think we don't need lines added 114-136.

Reply:
This is done.